### PR TITLE
[No QA] Update Payment Section of Contributing Guidelines

### PR DESCRIPTION
**Pull request by marcaaron**
### Details

Coming from [this Slack post](https://expensify.slack.com/archives/C01SKUP7QR0/p1657827530015949)

### Fixed Issues

No Issue

### Tests

No QA

### CODE REVIEW

File: contributingGuides/CONTRIBUTING.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Payment for your contributions will be made no less than 7 days after the pull r
 
 New contributors are limited to working on one job at a time, however experienced contributors may work on numerous jobs simultaneously. 
 
+Please be aware that compensation for any support in solving an issue is provided **entirely at Expensify’s discretion**. Personal time or resources applied towards investigating a proposal **will not guarantee compensation**. Compensation is only guaranteed to those who **propose a solution and get hired for that job**. We understand there may be cases where a selected proposal may take inspiration from a previous proposal. Unfortunately, it’s not possible for us to evaluate every individual case and we have no process that can efficiently do so. Issues with higher rewards come with higher risk factors so try to keep things civil and make the best proposal you can. Once again, **any information provided may not necessarily lead to you getting hired for that issue or compensated in any way.**
```
Comment (michaelhaxhiu) [inline contributingGuides/CONTRIBUTING.md:40]:
```suggestion
Please be aware that compensation for any support in solving an issue is provided **entirely at Expensify’s discretion**. Personal time or resources applied towards investigating a proposal **will not guarantee compensation**. Compensation is only guaranteed to those who **[propose a solution and get hired for that job](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#propose-a-solution-for-the-job)**. We understand there may be cases where a selected proposal may take inspiration from a previous proposal. Unfortunately, it’s not possible for us to evaluate every individual case and we have no process that can efficiently do so. Issues with higher rewards come with higher risk factors so try to keep things civil and make the best proposal you can. Once again, **any information provided may not necessarily lead to you getting hired for that issue or compensated in any way.**
```

**Review by michaelhaxhiu — COMMENTED** [review]
Thanks for suggesting the clarifications for future precedent.

File: contributingGuides/CONTRIBUTING.md [hunk 2]
```diff
@@ -37,6 +37,8 @@ Payment for your contributions will be made no less than 7 days after the pull r
 
 New contributors are limited to working on one job at a time, however experienced contributors may work on numerous jobs simultaneously. 
 
+Please be aware that compensation for any support in solving an issue is provided **entirely at Expensify’s discretion**. Personal time or resources applied towards investigating a proposal **will not guarantee compensation**. Compensation is only guaranteed to those who **[propose a solution and get hired for that job](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#propose-a-solution-for-the-job)**. We understand there may be cases where a selected proposal may take inspiration from a previous proposal. Unfortunately, it’s not possible for us to evaluate every individual case and we have no process that can efficiently do so. Issues with higher rewards come with higher risk factors so try to keep things civil and make the best proposal you can. Once again, **any information provided may not necessarily lead to you getting hired for that issue or compensated in any way.**
```
Comment (Beamanator) [inline contributingGuides/CONTRIBUTING.md:40]:
```suggestion
Please be aware that compensation for any support in solving an issue is provided **entirely at Expensify’s discretion**. Personal time or resources applied towards investigating a proposal **will not guarantee compensation**. Compensation is only guaranteed to those who **[propose a solution and get hired for that job](https://github.com/Expensify/App/blob/main/contributingGuides/CONTRIBUTING.md#propose-a-solution-for-the-job)**. We understand there may be cases where a selected proposal may take inspiration from a previous proposal. Unfortunately, it’s not possible for us to evaluate every individual case and we have no process that can efficiently do so. Issues with higher rewards come with higher risk factors so remember to keep discussions civil and make the best proposal you can. Once again, **providing any information may not necessarily lead to you getting hired for that issue or compensated in any way.**
```

**Review by Beamanator — APPROVED** [review]
Overall looks great, a few NAB suggested wording changes

**Pull request merged**